Docs/Sample.plist

```diff
@@ -1129,8 +1129,6 @@
 			<false/>
```

Comment:
Do we still implement CMD+OPT+P+R in OpenCore rather than in the driver?

Reply:
Okay... you're fine with an additional BEP feature to implement I guess arbitrary hotkeys?